Makefile

```diff
@@ -271,38 +279,47 @@ $(foreach prog,$(PROGRAMS),$(eval $(call BUILD_ASSEMBLY,$(prog))))
 
 ########################## MAKE/INSTALL RULES ##########################
 #
-default: dependencies core
+default: core
 
 core: game renderers mods utility
 
 tools: editor gamemonitor
 
-package: cli-dependencies core editor gamemonitor docs version
+package: all-dependencies core tools docs version
```

Comment:
First of all, we don't have any linux-exclusive bundled dependencies anymore. And the reason why this works is that the os-specific dependencies don't get copied into `packaging/built` by `make install`. OS X and Linux build a second tree based on `packaging/built` to create packages from, so their dependencies never end up in the "template", and NSIS cherry-picks the file it needs from `packaging/built`.